src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.services.DataProcessing;
+import core.basesyntax.services.FileDataReader;
+import core.basesyntax.services.FileDataWriter;
+import core.basesyntax.services.impl.DataProcessingImpl;
+import core.basesyntax.services.impl.FileDataReaderImpl;
+import core.basesyntax.services.impl.FileDataWriterImpl;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.FruitStrategyImpl;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.nio.file.Path;
+import java.util.List;
+
+public class Main {
+    private static final String inputPath = "src/main/java/core/basesyntax/resources/input.csv";

```

Comment:
```suggestion
    private static final String INPUT_PATH = "src/main/resources/input.csv";
```
use correct style for the constant. Pay attention  to the correct folder level